templates/components/common/share.html

```diff
@@ -61,11 +61,7 @@
             {{/each}}
         </ul>
         <script type="text/javascript"
-                src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
-        <script>
-            if (typeof(addthis) === "object") {
-                addthis.toolbox('.addthis_toolbox');
-            }
-        </script>
+                defer src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
+        <script type="text/javascript" defer src="/assets/js/theme/global/addthis.js"></script>
```

Comment:
Two things:
(1) If doing this, we should pass this path through the `cdn` helper to make sure we aren't hitting the site directly, but using the cdn.
(2) This will result in a second request which will probably hurt our time to interactive timing.

Take a look at https://www.addthis.com/academy/using-addthis-asynchronously/ for other techniques we can use.